site/zenodo_rdm/metrics/api.py

```diff
@@ -87,26 +87,27 @@ def get_uptime():
         url = current_app.config["METRICS_UPTIME_ROBOT_URL"]
         api_key = current_app.config["METRICS_UPTIME_ROBOT_API_KEY"]
 
-        end = datetime.utcnow().replace(
-            day=1, hour=0, minute=0, second=0, microsecond=0
-        )
-        start = (end - timedelta(days=1)).replace(day=1)
-        end_ts = calendar.timegm(end.utctimetuple())
-        start_ts = calendar.timegm(start.utctimetuple())
-
-        res = requests.post(
-            url,
-            json={
-                "api_key": api_key,
-                "custom_uptime_ranges": f"{start_ts}_{end_ts}",
-            },
-        )
+        if api_key:
+            end = datetime.now(timezone.utc).replace(
```

Comment:
I would put this change from `utcnow` to `now` as a separated commit, since it's unrelated and the indentation change can make it difficult to notice depending on the diffing tool used.